Content.Server/Salvage/SalvageSystem.ExpeditionConsole.cs

```diff
@@ -85,6 +118,66 @@ private void OnSalvageClaimMessage(EntityUid uid, SalvageExpeditionConsoleCompon
         UpdateConsoles(data);
     }
 
+    // Frontier
+    private void OnSalvageFinishMessage(EntityUid entity, SalvageExpeditionConsoleComponent component, FinishSalvageMessage e) // Frontier
+    {
+        if (!TryComp<SalvageExpeditionDataComponent>(_station.GetOwningStation(entity), out var data) || !data.CanFinish)
+            return;
+
+        // Based on SalvageSystem.Runner:OnConsoleFTLAttempt
+        if (!TryComp(entity, out TransformComponent? xform)) // Get the console's grid (if you move it, rip you)
+        {
+            PlayDenySound(entity, component);
+            _popupSystem.PopupEntity(Loc.GetString("salvage-expedition-shuttle-not-found"), entity, PopupType.MediumCaution);
+            UpdateConsoles(data);
+            return;
+        }
+
+        var query = EntityQueryEnumerator<HumanoidAppearanceComponent, TransformComponent>();
```

Comment:
This still mark cultists and etc as crew

Reply:
Pushed a newer version.  Seems alright, but it's got a fair bit in it.

Anything that can have a mind gets checked.  If nothing's around (ghosted?), if they have a bank account (and thus something to lose), you can't leave them behind - this may need revision, as a ghosted player might not need to be dragged about.

If there _is_ a mind, you then need to check if the mind is a hostile ghost role (space dragons and the like) - these should be ignored.  Otherwise, just make sure they're on the ship, alive or dead.